app/assets/stylesheets/custom.scss

```diff
@@ -155,6 +157,55 @@ th {
   text-align: center;
 }
 
+// about page
+
+.about {
+  background-color: #e3e3e3;
+  border: 2px solid #6699ff;
+  border-radius: 20px;
+  margin-bottom: 2px;
+  min-height: 20px;
```

Comment:
Properties should be ordered background-color, border, border-radius, margin-bottom, margin-left, margin-right, min-height, padding, width